accPlot.py

```diff
@@ -101,7 +101,10 @@ def plotTimeSeries(
     """
 
     # read time series file to pandas DataFrame
-    d = accUtils.loadTimeSeriesCSV(tsFile)
+    d = pd.read_csv(
+        tsFile, index_col='time',
+        parse_dates=['time'], date_parser=accUtils.date_parser
+    )
```

Comment:
No longer need specific loading function required to parse through the column names to determine the time column

Reply:
Hi @chanshing - given we will no longer support it, can you delete the loadTimeSeriesCSV() method from accUtils.py ?